src/onegov/reservation/models/custom_reservation.py

```diff
@@ -7,7 +7,7 @@
 
 
 class CustomReservation(Reservation, ModelBase, Payable):
-    __mapper_args__ = {'polymorphic_identity': 'custom'}  # type:ignore
+    __mapper_args__ = {'polymorphic_identity': 'custom'}
```

Comment:
Another merge conflict resolution error, this ignore is necessary on latest libres version.